### ISSUE TITLE
predict_abdomenatlas.py: pad_to_training_size pads the wrong axis on undersized inputs

### ISSUE THREAD

**Issue by sidd462** [MrGiovanni/R-Super#7]
## Summary

`predict_abdomenatlas.py` silently drops any input volume whose
post-preprocessing shape is smaller than the model's training size on
the z- or x-axis. No `predictions/*.nii.gz` outputs are produced for
the affected cases, so any downstream evaluator that globs for
prediction outputs will not see them.

## Symptom

For affected cases the script prints:

```
FAILED postprocess
```

with no further context (a bare-except in the postprocess block
swallows the error message and traceback).

## Root cause

The two `F.pad(...)` calls inside `pad_to_training_size()` have the
z- and x-axis padding tuples swapped.

`torch.nn.functional.pad` reads its padding tuple **last-dim-first**, so
for a 5D tensor of shape `(N, C, D, H, W)` the tuple must be ordered:

```
(W_left, W_right, H_left, H_right, D_left, D_right)
```

In the current code, the z-axis branch passes `(diff, diff, 0,0, 0,0)`
(which pads W, not D), and the x-axis branch passes
`(0,0, 0,0, diff, diff)` (which pads D, not W). When the input is too
small on the z-axis the code pads the x-axis instead (and vice versa).
The originally-too-small axis is left untouched and the next layer
rejects the resulting shape.

Relevant lines: L256 (z-axis branch) and L274 (x-axis branch) of
`predict_abdomenatlas.py`.

## Reproduction

The bug triggers on any input volume whose post-preprocessing shape is
`< 128` on the z- or x-axis (i.e. enters the padding branch in
`pad_to_training_size`). The padded tensor has the wrong axis enlarged
and is rejected by the next layer; the bare-except above turns this
into the silent `FAILED postprocess` print.
